dev_guide/downward_api.adoc

```diff
@@ -94,3 +94,97 @@ $ oc logs -p dapi-env-test-pod
 ====
 
 You should see `MY_POD_NAME` and `MY_POD_NAMESPACE` in the logs.
+
+== Consuming via volume plugin
+
+Another mechanism for consuming the downward API is the volume plug-in.  The downward API volume
+plug-in creates a volume with configured fields projected into files.  The `metadata` field of the
+`*VolumeSource*` API resource is used to configure this volume.  The plug-in supports the following
+fields:
+
+<1>  Pod name
+<2>  Pod namespace
+<3>  Pod annotations
+<4>  Pod labels
+
+Let's look at a small example of how the volume plug-in is configured:
+
+[source,yaml]
+====
+----
+spec:
+  volumes:
+    - name: podinfo
+      metadata: <1>
+        items:  <2>
+          - name: "labels" <3>
+            fieldRef:
+              fieldPath: metadata.labels <4>
+----
+
+<1> The `metadata` field of the volume source configures the downward API volume
+<2> The `items` field holds a list of fields to project into the volume
+<3> The name of the file to project the field into
+<2> The selector of the field to project 
+====
+
+Let's take a look at a full example:
+
+. Create a `*_volume-pod.json_*` file:
+
+[source,yaml]
+----
+kind: Pod
+apiVersion: v1
+metadata:
+  labels:
+    zone: us-east-coast
+    cluster: downward-api-test-cluster1
+    rack: rack-123
+  name: dapi-volume-test-pod
+  annotations:
+    annotation1: 345
+    annotation2: 456 
+spec:
+  containers:
+    - name: volume-test-container
+      image: gcr.io/google_containers/busybox
+      command: ["sh", "-c", "cat /etc/labels /etc/annotations"]
```

Comment:
Although you're showing that location here, can you put that information about where those files will be places in the initial paragraph or somewhere at the end?